src/platform/web/ui/session/settings/SettingsView.js

```diff
@@ -90,6 +90,18 @@ export class SettingsView extends TemplateView {
             })
         );
 
+        settingNodes.push(
+            t.h3("Theme"),
```

Comment:
I still think it's not the best use of my time to support a second theme at this point, but you're also not the only person asking for a dark theme, so perhaps we could add it as an unsupported option. Could we add a subtle notice here? Perhaps the aforementioned `ThemeManager` could give a label and id to every theme returned to the settings view model, and for dark mode we set the label to `Dark theme (experimental, unsupported, may be broken)`

Reply:
Good idea. I went for just `(experimental)` because otherwise the `select` gets rather wide. Just let me know if that is not enough.

![Screenshot 2021-05-19 at 20 46 35](https://user-images.githubusercontent.com/1137962/118867373-50f12500-b8e3-11eb-87c6-a57bf401ec43.png)